apps/ml/css/custom.css

```diff
@@ -1,4 +1,20 @@
-/* */
+/*
+ * Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Thanks for pointing it out. Fixed with https://github.com/wso2/carbon-ml/pull/245/commits/d78f8e20a9f53dd6f35ce74ec8d5a12bbc19e50c